src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo.tsx

```diff
@@ -1,34 +1,25 @@
 import { Box, Serif } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
 import { SystemContextConsumer } from "Artsy"
-import { track } from "Artsy/Analytics"
-import * as Schema from "Artsy/Analytics/Schema"
 import React from "react"
-import { createFragmentContainer, graphql } from "react-relay"
 
 import { ArtworkSidebarAuctionPartnerInfo_artwork } from "__generated__/ArtworkSidebarAuctionPartnerInfo_artwork.graphql"
 
 export interface ArtworkSidebarAuctionPartnerInfoProps {
   artwork: ArtworkSidebarAuctionPartnerInfo_artwork
 }
 
-@track()
 export class ArtworkSidebarAuctionPartnerInfo extends React.Component<
   ArtworkSidebarAuctionPartnerInfoProps
 > {
-  @track(() => ({
-    context_module: Schema.ContextModule.Sidebar,
-    action_type: Schema.ActionType.Click,
-    subject: Schema.Subject.AuctionBuyerPremium,
-    type: Schema.Type.Link,
-  }))
   render() {
     const { partner, sale_artwork, sale } = this.props.artwork
     if (sale.is_closed) {
       return null
     }
     return (
       <SystemContextConsumer>
-        {({ mediator }) => (
```

Comment:
This was no longer used, which makes me think we can remove the `<SystemContextConsumer/>` and probably make this a function component instead of a class.